README.md

```diff
@@ -4,6 +4,11 @@
 
 
 # Homebridge
+[![Website](https://img.shields.io/website?down_color=red&down_message=offline&up_color=green&up_message=online&url=https%3A%2F%2Fhomebridge.io)](https://homebridge.io)
```

Comment:
Do we need "website: online"  badge?

Reply:
not necessarily, but gives it a link to it? I can take it off?

Reply:
We can link to the website from the logo, it's big and prominent and can use badges to other things, like link to the community chat, which is a common practice among many projects.

Something like this:

```
[![Slack Status](https://homebridge-slackin.glitch.me/badge.svg)](https://homebridge-slackin.glitch.me)
```

Which already exists on the page, but might make sense to bring it up top, along with the discord badge:

```
[![Discord Status](https://img.shields.io/discord/512378789297258528.svg?style=flat)](https://discord.gg/RcV7fa8)
```

Reply:
Sounds good to me!